args: ['python3', 'tulip.py', './tests/compiler_errors/fn_tests/unclosed_fn.tlp']
-------- stdout ----------
--------------------------
-------- stderr ----------
./tests/compiler_errors/fn_tests/unclosed_fn.tlp:2:9 [ERROR]: Unclosed Function definition. Expected `END`, but found end of file instead
--------------------------
exit code: 1
